Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.h

```diff
@@ -234,14 +230,15 @@ namespace AzToolsFramework
         void SetRecursiveExpansionOngoing(bool isExpanding);
 
         // shared pools of recycled widgets
-        auto GetRowPool()
+        static auto GetRowPool()
         {
-            return m_rowPool;
+            return static_cast<AZ::InstancePoolManager*>(AZ::Interface<AZ::InstancePoolManagerInterface>::Get())->GetPool<DPERowWidget>();
         }
 
-        auto GetLabelPool()
+        static auto GetLabelPool()
         {
-            return m_labelPool;
+            return static_cast<AZ::InstancePoolManager*>(AZ::Interface<AZ::InstancePoolManagerInterface>::Get())
+                ->GetPool<AzQtComponents::ElidingLabel>();
```

Comment:
To be honest here.
InstancePoolManagerInterface isn't actually providing an interface since it has zero functions and is useless.

This code has a smell because of the need for a static_cast to access the `AZ::InstancePoolManager*`.

The better solution is to remove the `InstancePoolManagerInterface`, default the constructor and destructor of the `InstancePoolManager` class and then make a type alias for `InstancePoolManagerRegistrar` that points to `typename AZ::Interface<InstancePoolManager>::Registrar`
This code would then just directly access the InstancePool vfrom the AZ::InstancePoolManager that is registered with the global AZ::Interface.

Here is a link to the implementation change need for the the [InstancePoolManager](https://github.com/o3de/o3de/issues/14109)

This code should then update to the following
```c++
        static auto GetRowPool()
        {
            return AZ::Interface<AZ::InstancePoolInterfaceManager>::Get()->GetPool<DPERowWidget>();
        }

        static auto GetLabelPool()
        {
            return AZ::Interface<AZ::InstancePoolInterfaceManager>::Get()->GetPool<AzQtComponents::ElidingLabel>();
        }
```